### ISSUE TITLE
BUG: unable to move objects without repositioning the objects 

### ISSUE THREAD

**Issue by Fkaule** [zsviczian/obsidian-excalidraw-plugin#1312]
**Your environment**
SYSTEM INFO:
	Obsidian version: v1.4.5
	Installer version: v1.3.5
	Operating system: Darwin Kernel Version 22.6.0: Wed Jul  5 22:22:05 PDT 2023; root:xnu-8796.141.3~6/RELEASE_ARM64_T6000 22.6.0
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: AnuPpuccin v1.4.1
	Snippets enabled: 1
	Restricted mode: off
	Plugins installed: 28
	Plugins enabled: 27
		1: Calendar v1.5.10
		2: Editor Syntax Highlight v0.1.3
		3: Excalidraw v1.9.12
		4: Readwise Official v2.0.1
		5: Templater v1.16.0
		6: Settings Search v1.3.8
		7: Recent Files v1.3.7
		8: Dataview v0.5.56
		9: Style Settings v1.0.6
		10: Minimal Theme Settings v6.3.2
		11: Checklist v2.2.12
		12: Tasks v4.3.0
		13: Quick Switcher++ v3.3.4
		14: Ozan's Image in Editor Plugin v2.2.3
		15: Latex Suite v1.7.3
		16: Folder Note Core v1.3.5
		17: AidenLx's Folder Note v0.16.5
		18: Periodic Notes v0.0.17
		19: Convert url to preview (iframe) v0.5.0
		20: Supercharged Links v0.9.7
		21: Outliner v4.7.1
		22: ExcaliBrain v0.2.2
		23: Advanced URI v1.36.4
		24: Quiet Outline v0.3.15
		25: Zotero Integration v3.0.7
		26: Execute Code v1.8.1
		27: Commander v0.5.0

**Describe the bug**
When an object (e.g. a LaTeX Symbol) is attached to another object, I can't move the LaTeX Symbol without destroying my other objects. I know I can move objects without snapping with the modifier key, but when it's already snapped I can't do that.

**To Reproduce**
I tried to reconstruct the error with a small example, it's actually not that easy. But for my "actual" drawing, this allways happens (like in the gif below)

**Expected behavior**
It should be possible to move objects without moving other objects because of this. Maybe a fast solution would be to toggle the snapping off

**Screenshots**
![CleanShot 2023-09-06 at 11 24 31](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/69110042/017c8a89-42ad-4944-9b50-8eb2cc7838f4)

**Comment by zsviczian** [comment]
Yes, this is a frustrating problem. Try using the "Toggle to invert default binding behavior" feature from the command palette. What it does is to change the default behavior of arrow binding such that you need to hold down CTRL to bind, instead of needing to hold down CTRL to avoid binding when placing an object.

I also expect to see a change in the binding behavior relatively soon, where objects won't bind to arrows, only arrows to objects. There was a recent X poll about this here: https://twitter.com/excalidraw/status/1697286294890537046?s=20

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/14358394/ad4abe33-414a-4ef4-86b8-188602ce8845)

**Comment by Fkaule** [comment]
It would be really helpful to set these setting (Toggle to invert default binding behavior) as global setting. Now I have to do this every time I'm opening the drawing again.
